src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
+$day-size: 100px;
+$gap: 1px;
+$days: (
+  mon 0,
+  tue 101px,
+  wed 202px,
+  thu 303px,
+  fri 404px,
+  sat 505px,
+  sun 606px
+);
+
+.body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
```

Comment:
Move each BEM block to a separate file
Sam fo variables (according to BEM rules)